modyn/tests/trainer_server/internal/trainer/test_pytorch_trainer.py

```diff
@@ -258,6 +258,8 @@ def get_training_info(
                 label_transformer=PythonString(value=get_mock_label_transformer() if transform_label else ""),
                 grad_scaler_configuration=JsonString(value=json.dumps({})),
                 epochs_per_trigger=1,
+                grad_norm=0.1,
+                gradient_accumulation_steps=1,
```

Comment:
Could you add a test that actually semantically checks the gradient accumulation logic?